packages/builders/src/interactions/slashCommands/mixins/ApplicationCommandOptionWithChoicesAndAutocompleteMixin.ts

```diff
@@ -51,19 +54,19 @@ export class ApplicationCommandOptionWithChoicesAndAutocompleteMixin<T extends s
 	 *
 	 * @param choices The choices to add
 	 */
-	public addChoices(choices: [name: string, value: T][]): Omit<this, 'setAutocomplete'> {
+	public addChoices(...choices: APIApplicationCommandOptionChoice<T>[]): Omit<this, 'setAutocomplete'> {
 		if (this.autocomplete) {
 			throw new RangeError('Autocomplete and choices are mutually exclusive to each other.');
 		}
 
 		choicesPredicate.parse(choices);
 
-		for (const [label, value] of choices) this.addChoice(label, value);
+		for (const { name, value } of choices) this.addChoice({ name, value });
```

Comment:
Useless destructuring
```suggestion
		for (const entry of choices) this.addChoice(entry);
```